Python/codecs.c

```diff
@@ -50,6 +50,26 @@ int PyCodec_Register(PyObject *search_function)
     return -1;
 }
 
+int
+PyCodec_Unregister(PyObject *search_function)
+{
+    PyInterpreterState *interp = PyInterpreterState_Get();
+    /* Do nothing if codec_search_path is not created yet or was cleared. */
```

Comment:
Can we declare and reuse the variable?
```
PyObject *codec_search_path = interp->codec_search_path;
```

Reply:
Hm, I checked other functions of codecs use `interp->codec_search_path` directly, so I think it's fine to keep it ;)

Reply:
`PyObject *codec_search_path = interp->codec_search_path;` sounds like a good idea, since the function modifies the list in place.

Reply:
ok, updated, thanks:)